### PR TITLE
CORE-6332: enable E2E tests for ARM64 architecture

### PR DESCRIPTION
**Pull request by wzur-r3**
* added nodeSelector to the bootstrap jobs
* bootstrap jobs have their own bootstrap.nodeSelector in `values.yaml`,
  but failback to the `nodeSelector` used by the rest of deployments
* bootstrap jobs
* updated E2E pipeline to start workers on requested architecture
* E2E pipeline can start tests for AMD64 or ARM64 arch, selected by a
  new build parameter `BUILD_ARCH`, if not selected, default to AMD64
* fixed `sanitizedBranchName` to handle `-` at the end of branch name
* removed trailing whitespaces
* activated E2E tests for ARM64 nightly builds
* strip indents for all `sh` steps
* replaced triple double quote with triple single quote and thus use
  shell variable interpolation
* use `"${VAR_NAME}"` in all `sh` steps

A test nightly build is at https://ci02.dev.r3.com/blue/organizations/jenkins/Corda5%252FNightlys%252FCorda-Runtime-OS%252FLinux-ARM64/detail/PR-1976/2

### CODE REVIEW

File: .ci/e2eTests/Jenkinsfile [hunk 1]
```diff
@@ -37,6 +77,7 @@ pipeline {
         HELM_CONFIG_HOME = "/tmp/helm"
         HELM_REPOSITORY_CACHE = "/host_tmp/helm/repository"
         GRADLE_PERFORMANCE_TUNING = "--parallel -Dscan.tag.E2E -Dscan.tag.${env.NAMESPACE} --build-cache"
+        E2EWORKERARCH = "${e2eArchitecture}"
```
Comment (yarivus) [inline .ci/e2eTests/Jenkinsfile]: Maybe more readable to have: E2E_WORKER_ARCH

**Review by yarivus — CHANGES_REQUESTED** [review]

**Comment by wzur-r3** [comment]
added @benmcmahon100 for Helm chart changes

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**